src/components/icon/icons/thickArrowDown.jsx

```diff
@@ -1,3 +1,4 @@
+// DEPRECATED, Please use ChevronDownThick instead
```

Comment:
Now this svg is included twice, I would propose just re-exporting it in icons/index.js